resistor_color/resistor_color_task.py

```diff
@@ -0,0 +1,20 @@
+# Task from https://exercism.org/tracks/python/exercises/resistor-color
+
+def color_code(color):
+    switch = {
+        "black": 0,
+        "brown": 1,
+        "red": 2,
+        "orange": 3,
+        "yellow": 4,
+        "green": 5,
+        "blue": 6,
+        "violet": 7,
+        "grey": 8,
+        "white": 9
```

Comment:
Dict should be in a separate constant outside of the function

Reply:
This was fixed, changes pushed